packages/@vue/cli/__tests__/__e2e__/args.spec.js

```diff
@@ -0,0 +1,14 @@
+const path = require('path')
+const execa = require('execa')
+
+const CLI_PATH = path.resolve(__dirname, '..', '..', 'bin', 'vue.js')
+
+const runAsync = async (args, options) => execa(CLI_PATH, args, options)
+
+test('suggests matching command', async () => {
+  const { code, stdout } = await runAsync(['confgi'], { reject: false })
+
+  // Assertions
+  expect(code).toBe(1)
+  expect(stdout).toContain('Did you mean config?')
+})
```

Comment:
Why create a new file?

Reply:
It is always better to have an abstraction layer between `e2e` and unit tests.

Reply:
I agree. If so, it's better to refactor all project tests folder. Only create a `__e2e` for this `args.spec.js` will make directory structure more complicated. I think consistency first.

Reply:
In that case, let's keep the directory structure as it is for now.